doc/Installation/CentOS-image.md

```diff
@@ -72,3 +104,5 @@ cd /opt/librenms
   - WebUI
     - username: librenms
     - password: `D32fwefwef`
+
+> NOTE: Again, we highly advice that you change all passwords on this image when you deploy it!!
```

Comment:
s/advice/advise/

Reply:
Only there or in the top too?

And if that is the only spellcheck correction... My english is better than I thought